frontend/src/pages/phone.page.tsx

```diff
@@ -30,13 +31,32 @@ const Phone: NextPage = () => {
   const relayNumberData = useRelayNumber();
   const [isInOnboarding, setIsInOnboarding] = useState<boolean>();
   const realPhoneData = useRealPhonesData();
+  // The user hasn't completed the onboarding yet if...
+  const isNotSetup =
+    // ...they haven't purchased the phone plan yet, or...
+    profile?.has_phone === false ||
```

Comment:
non-blocking: is `profile?.has_phone` a boolean value? Maybe this can just be `!profile?.has_phone`

Reply:
It is, but `!profile?.has_phone` is also `true` if `profile?.has_phone` is `undefined`, which it is if `profile` is `undefined`. Which means that that would result in the user getting redirected when we're still loading the profile.

Reply:
Ahh.. gotcha.